easybuild/easyconfigs/b/Bonito/Bonito-0.1.0-foss-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,103 @@
+easyblock = 'PythonBundle'
+
+name = 'Bonito'
+version = '0.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/nanoporetech/bonito'
+description = "Convolution Basecaller for Oxford Nanopore Reads"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+local_pymajver = '3'
+local_pyminver = '7'
+local_pymajminver = local_pymajver + local_pyminver
+
+dependencies = [
+    ('Python', '%s.%s.4' % (local_pymajver, local_pyminver)),
```

These local variables are not used anywhere else in the easyconfig, so I think that it is better to drop them
```suggestion
dependencies = [
    ('Python', '3.7.4'),
```